### PR TITLE
ref(ui): Added tooltip - Datascrubbers v2 - Part 8

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Type:** Refactor

You can read more about this refactor here: https://app.asana.com/0/1155078590577881/1158903109364929

Result: 
image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/contextData.jsx [hunk 1]
```diff
@@ -161,7 +161,6 @@ class ContextData extends React.Component {
           <span
             key="value"
             className={
-              (valueInfo.isString ? 'val-string' : '') +
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/contextData.jsx]: Are strings no longer shown in this box?

File: src/sentry/static/sentry/app/components/events/contexts/user/user.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import {removeFilterMaskedEntries} from 'app/components/events/interfaces/utils';
+import ContextBlock from 'app/components/events/contexts/contextBlockV2';
+import ErrorBoundary from 'app/components/errorBoundary';
+import KeyValueList from 'app/components/events/interfaces/keyValueList/keyValueList';
+import {defined} from 'app/utils';
+
+import getUserKnownData from './getUserKnownData';
+
+type Props = {
+  data?: UserType & {data?: any};
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/contexts/user/user.tsx]: Are we able to narrow this any?

**Comment by matejminar** [comment]
According to percy, we seem to be skipping ip address:
image
Also, do we want to show empty field with value None? (username and name in this case)

**Comment by priscilawebdev** [comment]
@matejminar thank you for your feedback 🙂 Sometimes we don't display all the fields, this happen when the field is null. Please see:

image

Regarding the `None`, I'll check.. thanks for pointing it out

**Comment by matejminar** [comment]
But the IP on the left baseline snapshot seems to be 127.0.0.1?

**Comment by priscilawebdev** [comment]
@matejminar good catch the `ip_address `switch option was really missing...now it should be ok ;-)

**Review by matejminar — APPROVED** [review]

**Review by billyvg — COMMENTED** [review]
Oops I had a pending review from yesterday I forgot to submit

**Comment by billyvg** [comment]
@priscilawebdev Was it intentional to change the sorting? it looks like it is now reverse alphabetical

**Pull request merged**